Lib/Session/Storage/Handler/CassandraSessionHandler.php

```diff
@@ -70,29 +70,20 @@ public function __construct(ClusterInterface $cluster, array $options, LoggerInt
         $this->logger = $logger ?: new NullLogger();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function open($savePath, $sessionName)
+    public function open(string $savePath, string $sessionName): bool
     {
         $this->connectToCluster();
         $this->prepareStatements();
 
         return true;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function close()
+    public function close(): bool
     {
         return true;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function destroy($sessionId)
+    public function destroy(string $sessionId): bool
     {
         $blobSessionId = new \Cassandra\Blob($sessionId);
```

Comment:
Since you seem to want to stick to php's variable naming:
```suggestion
    public function destroy(string $id): bool
    {
        $blobSessionId = new \Cassandra\Blob($id);
```